src/usr/bin/diffuse

```diff
@@ -6817,15 +6822,15 @@ class NotebookTab(Gtk.EventBox):
     def __init__(self, name, stock):
         Gtk.EventBox.__init__(self)
         self.set_visible_window(False)
-        hbox = Gtk.HBox.new(False, 0)
+        hbox = Gtk.Box.new(Gtk.Orientation.HORIZONTAL, 0)
         if stock is not None:
             image = Gtk.Image.new()
             image.set_from_stock(stock, Gtk.IconSize.MENU)
             hbox.pack_start(image, False, False, 5)
             image.show()
         self.label = label = Gtk.Label.new(name)
         # left justify the widget
-        label.set_alignment(0, 0.5)
+        label.set_halign(Gtk.Align.START)
```

Comment:
I am unsure about this one, it does seem to be exactly the same behaviour. Did you test it?

Reply:
I tested this and saw no difference. When checking the [docs](https://lazka.github.io/pgi-docs/Gtk-3.0/classes/Misc.html#Gtk.Misc.set_alignment) again, I noticed that there are two alternatives to `set_alignment`. Perhaps it is better to use `Gtk.Label.set_xalign()` and `Gtk.Label.set_yalign()` instead. I will commit this soon.

Reply:
Indeed `set_alignment` documentation says we have two replacement:
* `Gtk.Label`’s `set_xalign` and `set_yalign` (new in 3.16), which take a float between 0.0 and 1.0
* `Gtk.Widget`'s `set_halign` and `set_valign` which take a `Gtk.Align`

I don't know which is better actually... :neutral_face: 

That said, you replaced `set_alignment(0, 0.5)` by just `set_halign(Gtk.Align.START)`, does it mean `valign` is set to `CENTER` by default?

Reply:
Just saw you changed these lines, resolving this thread.